config.TEMPLATE.inc.php

```diff
@@ -196,9 +195,6 @@
 ; Default locale
 locale = en_US
 
-; Client output/input character set
-client_charset = utf-8
```

Comment:
Could you document the removed settings in `docs/RELEASE` and `docs/release-notes/README-3.4`? There may be some users who were customizing this setting and will be confused when it suddenly stops being effective.

Reply:
Done.